public/css/main.css

```diff
@@ -45,3 +58,8 @@ p {
 .cta:hover, .cta:focus {
 	box-shadow: rgb(157, 18, 60) 0 0 .2em;
 }
+
+.optout {
```

Comment:
This is specific to the `tracking` page. These instructions should thus not be in the `main.css` file, but in a separate, `tracking.css` file.